python/graphstorm/config/argument.py

```diff
@@ -824,26 +824,26 @@ def use_early_stop(self):
 
     ## RGCN only ##
     @property
-    def n_bases(self):
+    def num_bases(self):
         """ Number of bases used in RGCN weight
         """
         # pylint: disable=no-member
-        if hasattr(self, "_n_bases"):
-            assert isinstance(self._n_bases, int)
-            assert self._n_bases > 0 or self._n_bases == -1
-            return self._n_bases
-        # By default do not use n_bases
+        if hasattr(self, "_num_bases"):
+            assert isinstance(self._num_bases, int)
```

Comment:
Could you help to add the error message of this and the next line assertion? Thanks.